src/frontend/components/Felleskomponenter/Steg/Steg.tsx

```diff
@@ -123,6 +124,7 @@ const Steg: React.FC<ISteg> = ({ tittel, skjema, gåVidereCallback, children })
                 felt.validerOgSettFelt(felt.verdi);
             });
         }
+        setParams({ availableLanguages: [] }).then();
```

Comment:
Lurer på om vi burde pakke denne inn i en navngitt funksjon for å forklare hva vi gjør og hvorfor. Type `skjulSpråkvelger()` eller liknende